language/functional_tests/tests/testsuite/transaction_fee_distribution/test_txn_fee_dust_distribution.mvir

```diff
@@ -0,0 +1,25 @@
+// The fees collected is less than the number of validators
+
+//! account: vivian, 1000000, 0, validator
+//! account: valentina, 1000000, 0, validator
+//! sender: vivian
```

Comment:
Nit: writing this right before `main` makes it easier to see which transaction this is associated with.